packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -1018,6 +1025,100 @@ class SlackBridge {
 			}
 		}
 	}
+
+	/*
+	 https://api.slack.com/events/message/message_deleted
+	 */
+	processSlackMessageDeleted(slackMessage) {
+		if (slackMessage.previous_message) {
+			let rocketChannel = this.getRocketChannel(slackMessage);
+			let rocketUser = RocketChat.models.Users.findOneById('rocket.cat', { fields: { username: 1 } });
```

Comment:
Not sure what this line is suppose to do (was already here).  Always returns rocket.cat.  Will look into this in the future.